src/main/java/org/mockito/runners/VerboseMockitoJUnitRunner.java

```diff
@@ -34,7 +34,10 @@
  * Do you think it is useful or not? Drop us an email at mockito@googlegroups.com
  * <p>
  * Experimental implementation - will change in future!
+ *
+ * @deprecated as of 2.0. Use the new {@link org.mockito.runners.MockitoJUnitRunner.Strict} runner instead.
```

Comment:
@TimvdLippe cosmetics: 2.0.0 is used in other places, it should be used here too